### DISCUSSION TITLE
Community Q&A

### DISCUSSION THREAD

**Discussion by david3107** [dcodx/gitarmor#13] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by timurxyz** [comment]
For file operations I would recommend checking out and migrate to bun instead of using the ancient node
https://bun.sh/guides/write-file/basic
:)

**Reply by david3107** [reply]
Thanks @timurxyz, we will consider it for the next version, as more file operations (including analyzing the content will be included)
